src/js/franklin-dashboard.js

```diff
@@ -39,119 +38,151 @@ angular
     'ui.router',
     'franklin-dashboard.services',
     'mm.foundation'
-  ])
-  .constant('VERSION', packageJson.version)
-  .config(($authProvider, ENV, $stateProvider, $urlRouterProvider, toastrConfig,
-    $resourceProvider, $interpolateProvider) => {
-
-    $interpolateProvider.startSymbol('[[').endSymbol(']]');
-
-    //angular routing depending on login status
-    $stateProvider
-      .state('logged', {
-        url: '/dashboard',
-        templateUrl: 'dashboard/dashboard.html',
-        resolve: {
-          loginRequired: loginRequired
+  ]);
+
+//get github client id from franklin-api
+//TODO: should get this in dashboard backend
```

Comment:
you won't get this from the dashboard backend. To keep things simple, you'll always just get this from API. We aren't worried about the 2nd API call right now.